deps/mmtk_julia.version

```diff
@@ -2,5 +2,5 @@ MMTK_JULIA_BRANCH = master
 MMTK_JULIA_SHA1 = c9e046baf3a0d52fe75d6c8b28f6afd69b045d95
 MMTK_JULIA_GIT_URL := https://github.com/mmtk/mmtk-julia.git
 MMTK_JULIA_TAR_URL = https://github.com/mmtk/mmtk-julia/archive/refs/tags/v0.30.4.tar.gz
```

Comment:
Do we need to update this URL?

Reply:
No need. We could already build the moving/sticky variations of the binding from source. The only thing that changed in this PR was building with the binary from BinaryBuilder.